packages/backend-api/src/processing/index.ts

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 import { registerTask } from './util';
 
-import {heartbeatTask, ICommentActionData} from './tasks';
+import { heartbeatTask, ICommentActionData } from './tasks';
```

Comment:
when practical send cleanups like this in separate no-expected-effect pull requests